### PR TITLE
Juliagomes/update Installation section of readme

### PR DESCRIPTION
**Pull request by jgomes168**
Update installation section of README to avoid errors.

Successful installation in Colab notebook:
Screenshot 2024-08-19 at 6 49 00 PM

Failed installation in Colab notebook:
Screenshot 2024-08-19 at 6 49 58 PM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,8 +49,11 @@ We defer readers to the arxiv paper to see the effectiveness of these prompts ag
 
 ## Installation
 
-```bash
-$ guardrails hub install hub://arize/dataset_embeddings
+```python
+from guardrails import install
+install("hub://arize-ai/dataset_embeddings_guardrails", quiet=True, install_local_models=True)
```
Comment (CalebCourier) [inline README.md:54]: This doesn't look quite right.  The validator is registered as `arize/dataset_embeddings` in the `main.py` file which typically means the hub id should be `hub://arize/dataset_embeddings` unless there are changes elsewhere I'm missing.

Comment (jgomes168) [reply README.md:54]: Hmm in the [Colab](https://colab.research.google.com/github/Arize-ai/dataset-embeddings-guardrails/blob/main/validator/dataset_embeddings_guard.ipynb) this seems to work

Screenshot 2024-08-19 at 7 06 13 PM

Comment (jgomes168) [reply README.md:54]: The other version throws an error

Screenshot 2024-08-19 at 7 07 11 PM

Comment (CalebCourier) [reply README.md:54]: There may be a discrepancy between this and the hub index, let me check

Comment (CalebCourier) [reply README.md:54]: Ok, sure enough this is the id listed in the hub index.  It's not the normal pattern but it should work.

**Review by CalebCourier — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jgomes168) [reply README.md:54]: Ok, let me know if anything needs to be updated on my side!

**Pull request merged**